e2e/assets/vulnerable_rust_deps/Cargo.lock

```diff
@@ -0,0 +1,963 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Should we include this Cargo.lock file?

Reply:
My assumption was that this would make cargo not auto-update to fixed versions, but given that Cargo.toml specifies the vulnerable version explicitly, including this is not necessary. Removed it.